### ISSUE TITLE
Server crash when opening certain file

### ISSUE THREAD

**Issue by GeForceLegend** [shader-slang/slang-vscode-extension#5]
Server will crash when open `Source/RenderPasses/ReSTIRPTPass/TemporalReuse.cs.lang` from https://github.com/DQLin/ReSTIR_PT 

Server log on first start and restart: (log level have been set to `message`)
```
// Opened vscode with only above file open
[Trace - 09:31:52] Sending request 'initialize - (0)'.
[Trace - 09:31:52] Received response 'initialize - (0)' in 134ms.
[Trace - 09:31:52] Sending notification 'initialized'.
[Trace - 09:31:52] Sending notification 'textDocument/didOpen'.
[Trace - 09:31:52] Sending request 'textDocument/inlayHint - (1)'.
[Trace - 09:31:53] Sending notification '$/setTrace'.
[Trace - 09:31:53] Received request 'client/registerCapability - (999)'.
[Trace - 09:31:53] Sending response 'client/registerCapability - (999)'. Processing request took 0ms
[Trace - 09:31:53] Received request 'workspace/configuration - (4627)'.
[Trace - 09:31:53] Sending response 'workspace/configuration - (4627)'. Processing request took 0ms
[Trace - 09:31:53] Received request 'workspace/inlayHint/refresh - (0)'.
[Trace - 09:31:53] Sending response 'workspace/inlayHint/refresh - (0)'. Processing request took 1ms
[Info  - 09:31:53] Connection to server got closed. Server will restart.
true
// Sever crashed and restarting. After restart for 5 times, vscode does not restart it anymore.
[Trace - 09:31:53] Sending request 'initialize - (0)'.
[Trace - 09:31:53] Received response 'initialize - (0)' in 162ms.
[Trace - 09:31:53] Sending notification 'initialized'.
[Trace - 09:31:53] Sending notification 'textDocument/didOpen'.
[Trace - 09:31:53] Sending request 'textDocument/inlayHint - (1)'.
[Trace - 09:31:53] Received request 'client/registerCapability - (999)'.
[Trace - 09:31:53] Sending response 'client/registerCapability - (999)'. Processing request took 0ms
[Trace - 09:31:53] Received request 'workspace/configuration - (4627)'.
[Trace - 09:31:53] Sending response 'workspace/configuration - (4627)'. Processing request took 0ms
[Info  - 09:31:53] Connection to server got closed. Server will restart.
true
```

**Comment by csyonghe** [comment]
Fixed in extension v1.7.6, thanks for reporting the issue!

**Comment by HanBinGer** [comment]
Same problem on v1.8.19 with [PathBuilder.slang](https://github.com/DQLin/ReSTIR_PT/blob/master/Source/RenderPasses/ReSTIRPTPass/PathBuilder.slang).
Logs:
```
[Trace - 6:02:05 PM] Sending request 'initialize - (0)'.
[Trace - 6:02:05 PM] Received response 'initialize - (0)' in 204ms.
[Trace - 6:02:05 PM] Sending notification 'initialized'.
[Trace - 6:02:05 PM] Sending notification 'textDocument/didOpen'.
[Trace - 6:02:05 PM] Received request 'client/registerCapability - (999)'.
[Trace - 6:02:05 PM] Sending response 'client/registerCapability - (999)'. Processing request took 0ms
[Trace - 6:02:05 PM] Received request 'workspace/configuration - (4627)'.
[Trace - 6:02:05 PM] Sending response 'workspace/configuration - (4627)'. Processing request took 0ms
[Trace - 6:02:05 PM] Sending request 'textDocument/inlayHint - (1)'.
[Trace - 6:02:05 PM] Received request 'workspace/inlayHint/refresh - (0)'.
[Trace - 6:02:05 PM] Sending response 'workspace/inlayHint/refresh - (0)'. Processing request took 1ms
[Trace - 6:02:05 PM] Sending notification '$/setTrace'.
[Trace - 6:02:05 PM] Sending notification 'workspace/didChangeConfiguration'.
[Trace - 6:02:05 PM] Sending notification '$/setTrace'.
[Trace - 6:02:05 PM] Sending notification 'workspace/didChangeConfiguration'.
[Info  - 6:02:05 PM] Connection to server got closed. Server will restart.
true
[Trace - 6:02:05 PM] Sending request 'initialize - (0)'.
[Trace - 6:02:05 PM] Received response 'initialize - (0)' in 176ms.
[Trace - 6:02:05 PM] Sending notification 'initialized'.
[Trace - 6:02:05 PM] Sending notification 'textDocument/didOpen'.
[Trace - 6:02:05 PM] Received request 'client/registerCapability - (999)'.
[Trace - 6:02:05 PM] Sending response 'client/registerCapability - (999)'. Processing request took 0ms
[Trace - 6:02:05 PM] Received request 'workspace/configuration - (4627)'.
[Trace - 6:02:05 PM] Sending response 'workspace/configuration - (4627)'. Processing request took 0ms
[Trace - 6:02:05 PM] Sending request 'textDocument/inlayHint - (1)'.
[Trace - 6:02:05 PM] Received request 'workspace/inlayHint/refresh - (0)'.
[Trace - 6:02:05 PM] Sending response 'workspace/inlayHint/refresh - (0)'. Processing request took 0ms
[Info  - 6:02:05 PM] Connection to server got closed. Server will restart.
true
...
